src/pages/steps/AdditionalInformationStep/OfferScore.tsx

```diff
@@ -184,7 +191,7 @@ const OfferScore = ({ completedFields, offerId, scope, ...props }: Props) => {
     return maxRotation * scorePercentage + minRotation;
   }, [score]);
 
-  const tip = useMemo(() => {
+  const tipField = useMemo(() => {
     if (score === 100)
       return t(`create.additionalInformation.event_score.tip.completed`);
```

Comment:
This branch of the condition isn't used anymore I think

Reply:
is it possible this was an earlier commit?
It should already be `tipField`